sdk/azcore/policy/policy.go

```diff
@@ -19,6 +19,9 @@ import (
 // Request and react to the received Response.
 type Policy = exported.Policy
 
+// PolicyFunc is a type that implements the Policy interface. Use this type when implementing a stateless policy as a first-class function.
+type PolicyFunc = exported.PolicyFunc
```

Comment:
This type name stutters, it should probably be `Func`, making the fully qualified type name `policy.Func`.

Reply:
I think it will be ambiguous because we don't know from its name what Func does.

Reply:
The fully qualified name will be `policy.Func`.  The name will never be just `Func` unless it's referenced from within the same package.  Compared to `policy.PolicyFunc` which stuttered (i.e. the word "policy" is repeated).

Reply:
updated.